tests/codegen-llvm/simd/extract-insert-dyn.rs

```diff
@@ -25,97 +26,105 @@ pub struct u32x16([u32; 16]);
 pub struct i8x16([i8; 16]);
 
 // CHECK-LABEL: dyn_simd_extract
-// CHECK: extractelement <16 x i8> %x, i32 %idx
+// CHECK: extractelement <16 x i8> %[[TEMP:.+]], i32 %idx
```

Comment:
Out of curiosity, is this change necessary? If so, what is being generated here on RISC-V?